flutter_vlc_player/example/lib/vlc_player_with_controls.dart

```diff
@@ -1,19 +1,23 @@
+import 'dart:io';
 import 'dart:typed_data';
 
 import 'package:flutter/material.dart';
 import 'package:flutter/rendering.dart';
 import 'package:flutter_vlc_player/flutter_vlc_player.dart';
+import 'package:path_provider/path_provider.dart';
 
 import 'controls_overlay.dart';
 
 class VlcPlayerWithControls extends StatefulWidget {
   final VlcPlayerController controller;
   final bool showControls;
+  final Function(String) onStopRecording;
```

Comment:
Let's extract this callback into a typedef, and specify a return type. Right now it is inferred as `dynamic`, and it's unclear what exactly is returned.